content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,12 @@
 ---
-title: A Simple Component
+title: Een Eenvoudige Component
```

Comment:
Eenvoudig?

Reply:
@duivvv If you're commenting on the need for the additional "e", I think it's necessary here because of the Dutch word "component"'s genus.

Reply:
Looks very weird, but I trust you on this👍